### PR TITLE
Add django admin page for collections

### LINKED ISSUE #6418: add django admin pages to change collection contributors and owner
The new collection API won't publicly support contributors but we still have them for Mozilla controlled collections (e.g. featured addons/themes) so we need a django admin page to modify stuff.

### PR DESCRIPTION
**Pull request by eviljeff**
Fixes #6418

### CODE REVIEW

File: src/olympia/bandwagon/models.py [hunk 1]
```diff
@@ -450,9 +450,7 @@ def post_save_or_delete(sender, instance, **kw):
 class CollectionUser(models.Model):
     collection = models.ForeignKey(Collection)
     user = models.ForeignKey(UserProfile)
-    role = models.SmallIntegerField(
-        default=1,
-        choices=amo.COLLECTION_AUTHOR_CHOICES.items())
+    role = models.SmallIntegerField(default=1)  # TODO: drop me.
```
Comment (eviljeff) [inline src/olympia/bandwagon/models.py]: We'll be dropping a bunch of Collection models properties after frontend takes over so we can drop this one at the same time.

File: src/olympia/bandwagon/admin.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
+from django.contrib import admin
+
+from .models import Collection, CollectionUser
+
+
+class ContributorInline(admin.TabularInline):
+    model = CollectionUser
+    raw_id_fields = ('user',)
+    fields = ('user',)
+    verbose_name_plural = 'Contributors'
```
Comment (EnTeQuAk) [inline src/olympia/bandwagon/admin.py]: Let's localize all the things

File: src/olympia/bandwagon/models.py (hunk 1 above)
Comment (EnTeQuAk) [reply src/olympia/bandwagon/models.py]: Let's add a when, why and reference an github issue too please

File: src/olympia/bandwagon/admin.py (hunk 2 above)
Comment (eviljeff) [reply src/olympia/bandwagon/admin.py]: Sure.  Though a total of 3 people will see this in django admin on prod and they can all read English :)

Comment (eviljeff) [reply src/olympia/bandwagon/admin.py]: https://github.com/mozilla/addons-server/pull/6494/commits/3a06d116dbe80282f6ff2673678a0ad699f1baef

File: src/olympia/bandwagon/models.py (hunk 1 above)
Comment (eviljeff) [reply src/olympia/bandwagon/models.py]: https://github.com/mozilla/addons-server/pull/6494/commits/3a06d116dbe80282f6ff2673678a0ad699f1baef

**Pull request merged**